.github/workflows/test.yml

```diff
@@ -63,7 +64,7 @@ jobs:
         uses: "./.github/actions/unit_tests"
 
       - name: Run Lighthouse CI on the PWA
-        if: env.IS_DEFAULT_NPM == 'true'
+        if: env.IS_MRT_NODE == 'true'
```

Comment:
This is going to be true for the three NPM versions defined in the matrix, i.e. lithouse will run in 3 builds (https://github.com/SalesforceCommerceCloud/pwa-kit/blob/1df6b6711520b7bdbd5fdf96128f916cf269c1c5/.github/workflows/test.yml#L37)

Reply:
ah, good catch